Models/Product.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DFBangazon.Models
+{
+    public class Product
+    {
+        public int Id { get; set; }
+        public int Price { get; set; }
+        public string Title { get; set;}
+        public string ProductDesc { get; set; }
+        public int Quantity { get; set; }
+        public Customer CustomerId { get; set;}
```

Comment:
Check on the Order example - I think you will want to have both the int of the Customer Id but then the Customer of Customer. 
We will use the CustomerId to grab the customer info from the DB and then set them as the (type) Customer of Customer on the Product itself.

Probably same for ProductType.

On Order it looks like:
```
[Required]
    public int CustomerId { get; set; }
    public Customer Customer { get; set; }
```